src/Mjolnir.ts

```diff
@@ -658,4 +682,18 @@ export class Mjolnir {
             message: message /* If `undefined`, we'll use Synapse's default message. */
         });
     }
+
+    public queueRedactUserMessagesIn(userId: string, roomId: string) {
+        this.eventRedactionQueue.add(new RedactUserInRoom(userId, roomId));
+    }
+
+    /**
+     * Process all queued redactions, this is usually called at the end of the sync process,
```

Comment:
Could you detail a bit the error handling, e.g. `If some redaction cannot be processed, skip that redaction and process with other queued redactions.`

Also, if a redaction cannot be processed, is it removed from the queue?

Reply:
Currently we remove it from the queue if it fails, one of the reasons being that we would have to think about all of the situations where we wouldn't want failures to try again (e.g. messages were already redacted), although some of this might already be handled.

Reply:
Could you add this to the documentation?